### ISSUE TITLE
Where is TMCLParam.tmc ?

### ISSUE THREAD

**Issue by dzid26** [trinamic/TMCL-IDE-Nightly#11]
Firmware manuals mention **TMCLParam.tmc** is provided with IDE, but I cannot find it in TMCL-IDE folder. 
It would be best if it was natively included in the editor.

![image](https://user-images.githubusercontent.com/841061/71647953-d44caf00-2ccb-11ea-94ed-ed416a7cbbae.png)

**Comment by alpytron** [comment]
Any update on this?
I could not find that file either.

**Comment by trinamic-mj** [comment]
Ja, hab ich. Wollt auch grad ne Nightly bauen, komm aber nicht remote auf meinen Rechner.

Martin
________________________________
Von: alpytron <notifications@github.com>
Gesendet: Mittwoch, 11. November 2020 12:08
An: trinamic/TMCL-IDE-Nightly <TMCL-IDE-Nightly@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: Re: [trinamic/TMCL-IDE-Nightly] Low: Where is TMCLParam.tmc ? (#11)

EXTERNAL EMAIL



Any update on this?
I could not find that file either.

**Comment by trinamic-ed** [comment]
Wenn der aus ist, dan ist Marius / Hagen oder eventuel Joshua noch in der Firma und könnte bestimmt einschalten.

**Comment by alpytron** [comment]
Is it possible to just simply upload that file somewhere here in github? :)

**Comment by trinamic-mj** [comment]
Done

**Comment by alpytron** [comment]
Thank you, Martin.
This is a huge help for us.
